src/widgets/Components/NextCourse.tsx

```diff
@@ -126,15 +124,15 @@ const NextCourseLesson: React.FC<{
         if (days > 0) {
           setPrettyTime(`dans ${days} jour(s)`);
         } else if (hours > 0) {
-          setPrettyTime(`dans ${hours}h ${lz(minutes)}min`);
+          setPrettyTime(`dans ${hours}h ${minutes.toString().padStart(2, "0")}min`);
```

Comment:
C'est pas mieux des majuscules ?

```suggestion
          setPrettyTime(`Dans ${hours}h ${minutes.toString().padStart(2, "0")}min`);
```